examples/dmm/dmm.py

```diff
@@ -190,7 +190,7 @@ def model(self, mini_batch, mini_batch_reversed, mini_batch_mask,
         # this marks that each datapoint is conditionally independent of the others
         with pyro.plate("z_minibatch", len(mini_batch)):
             # sample the latents z and observed x's one time step at a time
-            for t in range(1, T_max + 1):
+            for t in pyro.markov(range(1, T_max + 1)):
```

Comment:
this change could use a comment

Reply:
some of the code snippets in dmm.ipynb would also need to be updated

Reply:
> some of the code snippets in dmm.ipynb would also need to be updated

I added a comment to the script.  I'll leave additional explanation/advertising to a followup PR.